site/en/blog/iframe-credentialless/index.md

```diff
@@ -0,0 +1,81 @@
+
+To enable cross-origin isolation, websites must send the following HTTP headers:  
+`Cross-Origin-Embedder-Policy: require-corp`  
+`Cross-Origin-Opener-Policy: same-origin`
+
+[COEP:credentialless](/blog/coep-credentialless-origin-trial/) can also be used as an alternative to `require-corp`. See [documentation](https://developer.mozilla.org/docs/Web/HTTP/Headers/Cross-Origin-Embedder-Policy) for further details.
+
+## Challenges with enabling COEP
+
+While cross-origin isolation brings webpages better security and the ability to enable powerful features, deploying COEP can be [difficult](https://web.dev/cross-origin-isolation-guide/). One of the biggest challenges is that all cross-origin iframes must deploy COEP and [CORP](https://web.dev/security-headers/#corp). Iframes without those headers will not be loaded by the browser.
+
+## Iframe credentialless to the rescue
+
+We're introducing iframe credentialless to help embed third-party iframes that don't set COEP. By adding the `credentialless` attribute to the `<iframe>` element, the iframe is loaded from a different, ephemeral storage partition and it isn't subject to COEP restrictions anymore. 
+
+Example:  
+`<iframe credentialless src="https://example.com">`
+
+Iframe is created in [a new ephemeral context](https://developer.mozilla.org/docs/Web/Security/IFrame_credentialless#the_solution_%E2%80%94_iframe_credentialless) and doesn't have access to any of the cookies associated with the top level website, because it starts from an empty cookie jar. Likewise, storage APIs such as [LocalStorage](https://developer.mozilla.org/docs/Web/API/Window/localStorage), [CacheStorage](https://developer.mozilla.org/docs/Web/API/CacheStorage), [IndexedDB](https://developer.mozilla.org/docs/Web/API/IndexedDB_API), and so on, are loading and storing data in the new ephemeral partition. The partition is scoped to the current top-level document and origin of the iframe. Storage will be cleared once the top-level document is unloaded.
+
+Iframes credentialless are not subject to COEP embedding rules. They are still secure, because they are loaded from a new empty context everytime. They are loaded without their data being personalized. They contain only public data, which is not valuable to an attacker.
+
+## Demo
+
+You can check out the [demo of an iframe credentialless](https://iframe-credentialless.glitch.me/).
+
+## FAQ
+
+**Will this feature be adopted by other browsers?**
```

Comment:
Should they be a sub-sub-sub title?
```text
### Will this feature be adopted by other browsers?
```

This would let developer extract a link to the section, for sharing. This might also lead to a better visual output.

Reply:
done